doc/es_ES/2fa.markdown

```diff
@@ -24,7 +24,7 @@ Instalacion
 2. Click a la izquierda en **Two factor authentication** y selecionar la caja
 3. Una key secret es geradada para ti
 
-![2FA](screenshots/2fa.png)
+![2FA](https://kanboard.net/screenshots/documentation/2fa.png)

```

Comment:
I'm sorry I have no idea why I did it, but you want me to modify it so that in the next version of kanboard comes out the whole translation